fractions/jaeger/src/main/java/org/wildfly/swarm/jaeger/deployment/JaegerInitializer.java

```diff
@@ -54,12 +58,11 @@ public void contextInitialized(ServletContextEvent servletContextEvent) {
 
         String serviceName = getProperty(sc, JAEGER_SERVICE_NAME);
         if (serviceName == null || serviceName.isEmpty()) {
-            logger.info("No Service Name set. Skipping initialization of the Jaeger Tracer.");
-            return;
+            logger.warn("No Service Name set. Using default. Please change it.");
+            serviceName = "thorntail/unknown";
```

Comment:
Can't you derive a better default from the context? For WildFly, we default to the deployment unit's name.

Reply:
+1, I just don't know how. @Ladicek @kenfinnigan any suggestions?

Reply:
For reference, this is how it's done in WildFly:
https://github.com/wildfly/wildfly/blob/75584e129564fbd13015f020fa9b9fe72d5e9c79/microprofile/opentracing-extension/src/main/java/org/wildfly/extension/microprofile/opentracing/TracingDeploymentProcessor.java#L114-L134

Reply:
I'm not sure if there's a way to get that from inside the deployment itself

Reply:
Then we cannot get anything better than the current name.

@kenfinnigan anything more to be done before approving/merging?

Reply:
@kenfinnigan can this be merged?